alert-common/src/main/java/com/synopsys/integration/alert/common/rest/api/ConfigurationCRUDHelper.java

```diff
@@ -87,7 +87,7 @@ public ConfigurationCRUDHelper(AuthorizationManager authorizationManager) {
         }
     }
 
-    public <T> ActionResponse<T> delete(BooleanSupplier existingModelSupplier, Runnable deleteFunction, ConfigContextEnum context, DescriptorKey descriptorKey) {
+    public <T> ActionResponse<T> delete(BooleanSupplier existingModelSupplier, Runnable deleteFunction) {
```

Comment:
This is a little trickier since Java doesn't give us a functional interface equivalent to Runnable-- in Jenkins Common we have a ThrowingExecutor-- maybe we should have something similar in Alert Common? Executor may not be the best name since Executors are a thing in Java that run Runnables.